### DISCUSSION TITLE
15.1.3

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#699] [General]
## Releases Notes for 15.1.3
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/draw.io-15.1.3-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/draw.io-15.1.3-windows-no-installer.exe)
[macOS - Intel](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/draw.io-x64-15.1.3.dmg)
[macOS - ARM](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/draw.io-arm64-15.1.3.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/drawio-amd64-15.1.3.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/drawio-x86_64-15.1.3.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v15.1.3/drawio-x86_64-15.1.3.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [diagrams.net core 15.1.3](https://github.com/jgraph/drawio/blob/v15.1.3/ChangeLog). All changes from 14.9.7 to 15.1.3, inclusive, are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v15.1.3'>15.1.3</a>.</em>

**Comment by jakemdaly** [comment]
I updated to this version and the GUI super slow and laggy (almost unusable). I tried uninstalling and reinstalling, but no luck. Switched back to 14.9.6 and it seems so be working fine as before

**Reply by davidjgraph** [reply]
@jakemdaly what are your environment details please? Which build did you use?

**Reply by GrahamChastney** [reply]
I'm experiencing the same, very slow. Minutes to start up and multi-seconds between any diagramming activity. Windows 10.

**Reply by davidjgraph** [reply]
Sorry, I mean full environment details. OS build, hardware spec, particularly graphics card, anything like a virus scanner that might be intercepting processes on the fly, etc.

**Reply by davidjgraph** [reply]
Worth trying 15.2.5. Are you sure you've the right 32/64 bit version running?

**Reply by jakemdaly** [reply]
OS - Microsoft Windows 10 Enterprise
Version - 10.0.18363
System - HP EliteBook 745 G6
Processor - AMD Ryzen 7 PRO 3700U w/ Radeon Vega Mobile
Yes I have the 64 bit version

**Reply by davidjgraph** [reply]
@jakemdaly no improvement in 15.2.7?

**Comment by demarb** [comment]
Hello. I used to use the desktop version of this app a while back but I stopped because it had a bug that wouldn't allow me to export files in other formats to my local storage. It would give an error occurred whilst saving this file. I recently used the app again but I updated it before doing so to version 15.1.3 that was released yesterday. But I am still having the issue so I just wanted to bring it to your attention. I ended up having to open the file in your web app, so I could export it.

I'm using a Dell Inspiron 15 3000 series laptop with 8GB ram and an AMD A6 7th gen processor. My current virus scanner is AVG free version. I don't think it is blocking it.

Thank you

**Reply by davidjgraph** [reply]
Could you start the machine in safe mode please? We would need it tested with the processes normally started at startup left off. Just export then work correctly?
